README.md

```
@@ -251,453 +258,110 @@ receiver := parkinglot.NewReceiver(client,
+commit order, so publish-time writes never block on the broker. Two relay
+runtimes share one publish-side API. On **TiDB**, a leader-elected sequencer
+pass assigns a dense, gapless offset (`seq`) to committed rows after the
+fact, so a transaction that started earlier but committed later is never
+skipped. On **MongoDB**, a relay tails a change stream on the insert-only
+outbox collection, reusing the oplog's total commit order — no sequencer
+needed. Both give **at-least-once** delivery in commit order (equivalent to
+one Kafka partition) and support independent consumer groups, each with its
+own offset/resume token; a new group starts at "latest" (future events only)
+by default.
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)